src/screens/profile/components/info-section/info-section.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { View } from 'react-native';
+
+import { TagType } from 'src/common/enums';
+import { Tag, Text } from 'src/components';
+import styles from './styles';
+
+type InfoSectionProps = {
+  header: string;
+  tags: string[];
+};
+
+const InfoSection: React.FC<InfoSectionProps> = ({ header, tags }) => {
+  return (
+    <View>
+      <Text style={styles.heading}>{header}</Text>
+      <View style={styles.tagsContent}>
+        {tags.map((item, index) => {
+          return (
+            <Tag
+              style={styles.tag}
+              tagType={TagType.COMMON}
+              key={index}
```

Comment:
Let's use `item` as a key. Or at least left `TODO` mark to switch to tagId once we have real data